reconcile/test/ocm/test_utils_ocm_service_log.py

```diff
@@ -0,0 +1,224 @@
+from datetime import (
+    datetime,
+    timedelta,
+    timezone,
+)
+
+import httpretty as httpretty_module
+import pytest
+from pytest_mock import MockerFixture
+
+from reconcile.test.ocm.conftest import (
+    register_ocm_get_list_request,
+    register_ocm_post_request,
+)
+from reconcile.utils.ocm import service_log
+from reconcile.utils.ocm.search_filters import (
+    DateRangeCondition,
+    Filter,
+)
+from reconcile.utils.ocm.service_log import (
+    OCMClusterServiceLog,
+    OCMServiceLogSeverity,
+    create_service_log,
+    get_service_logs,
+)
+from reconcile.utils.ocm_base_client import OCMBaseClient
+
+
+def build_service_log(
+    summary: str = "",
+    description: str = "",
+    cluster_uuid: str = "",
+    service_name: str = "some-service",
+    severity: OCMServiceLogSeverity = OCMServiceLogSeverity.Info,
+    timestamp: datetime = datetime(2020, 1, 2, 0, 0, 0, 0, tzinfo=timezone.utc),
+) -> OCMClusterServiceLog:
+    return OCMClusterServiceLog(
+        id="",
+        href="",
+        event_stream_id="",
+        username="",
+        cluster_uuid=cluster_uuid,
+        cluster_id="",
+        service_name=service_name,
+        severity=severity,
+        description=description,
+        summary=summary,
+        timestamp=timestamp,
+    )
+
+
+@pytest.fixture
+def example_service_log(
+    ocm_api: OCMBaseClient, httpretty: httpretty_module
+) -> OCMClusterServiceLog:
+    expected_service_log = build_service_log(
+        "some error",
+        "description",
+        "cluster_uuid",
+        severity=OCMServiceLogSeverity.Error,
+    )
+    register_ocm_get_list_request(
+        ocm_api,
+        httpretty,
+        "/api/service_logs/v1/cluster_logs",
+        [expected_service_log],
+    )
+    return expected_service_log
+
+
+def test_get_service_logs(
+    ocm_api: OCMBaseClient,
+    httpretty: httpretty_module,
```

Comment:
You don't need to specify the `httpretty` fixture because it's already loaded via `example_service_log`

Reply:
thanks. leftover from restructuring. i removed it.